jmclient/jmclient/wallet.py

```diff
@@ -1136,7 +1137,7 @@ def get_script_path(self, path):
 
     def get_path(self, mixdepth=None, internal=None, index=None):
         if mixdepth is not None:
-            assert isinstance(mixdepth, int)
```

Comment:
Can confirm that I received another error from this line as well when filling an offer. I'll pull down this branch and install it.